src/integrationTest/java/uk/gov/hmcts/reform/pdf/service/integration/GeneratedPDFContentV2Test.java

```diff
@@ -47,27 +64,126 @@ private static String textContentOf(byte[] pdfData) throws IOException {
     @SuppressWarnings("unchecked")
     @Test
     public void shouldCreateExpectedPdfFromPlainHtmlTemplate() throws Exception {
-        assertThatGeneratedPdfContains("<html><body>Hello!</body></html>", Collections.EMPTY_MAP, "Hello!");
+        Mockito.when(authorisationService.authorise(Mockito.anyString())).thenReturn("test-service");
+
+        MockHttpServletResponse response = getResponse(getRequestWithAuthHeader(
+            "<html><body>Hello!</body></html>",
+            Collections.emptyMap()
+        ));
+
+        assertThat(textContentOf(response.getContentAsByteArray())).contains("Hello!");
+        Mockito.verify(converter, Mockito.times(1))
+            .convert(Mockito.any(), Mockito.anyMapOf(String.class, Object.class));
```

Comment:
Why is it necessary to verify calling the converter?

Reply:
just to be sure we are doing the right thing